build/typescript-model/generate.sh

```diff
@@ -53,12 +53,49 @@ EOF
     export OPENAPI_GENERATOR_COMMIT="v6.3.0"
     bash $WORK_DIR/gen/openapi/typescript.sh $WORK_DIR/typescript-models $WORK_DIR/config.sh
 
-    apply_sed 's/\"name\": \".*\"/"name": "@devfile\/api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"description\": \".*\"/"description": "Typescript types for devfile api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"repository\": \".*\"/"repository": "devfile\/api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"license\": \".*\"/"license": "Apache-2.0"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"@types\/bluebird\": \".*\"/"@types\/bluebird": "3.5.21"/g' $WORK_DIR/typescript-models/package.json
-    
+    local workdir=$(pwd)
+    cd "$WORK_DIR/typescript-models"
+
+    ######################################################################################################
+    echo "[INFO] preparing package.json"
+    ######################################################################################################
+
+    echo "$(jq '. += {"name": "@devfile/api"}' package.json)" > package.json
```

Comment:
With the addition of `jq` to this script I wonder if it makes sense to include it to the list of [prerequisites](https://github.com/devfile/api/blob/main/CONTRIBUTING.md#prerequisites)? Also `node` though I feel like this should of been there before for generating typescript models.

Reply:
If we speak about the action that builds and then publishes the typescript models, it runs on ubuntu and there is a dedicated section to install node https://github.com/devfile/api/blob/main/.github/workflows/release-typescript-models.yaml#L48-L53. So no issues with it.

If you want to build typescript models locally, you definitely need to have `jq` and `node` installed.
I think it make sense to add both `jq` and `node` to the list of prerequisites (I'll do it)

Btw, typescript models are generated in a separate container. Node is used then to build and publish the generated library.

Reply:
@vitaliy-guliy Sounds good :+1:

Reply:
@michael-valdron I've added two items to the prerequisites. Could you take a look?

Reply:
And please merge the PR if everything's OK.
I don't have enough rights to do this.